prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs

```diff
@@ -383,5 +383,13 @@ public async Task<string> GetLinkedGpidAsync(int enrolleeId)
 
             return linkedGpid;
         }
+
+        public async Task<bool> IsAlwaysManualPaperEnrolment(int paperEnrolmentId)
```

Comment:
Since this does no check to validate it's a paper enrolment you want to either change the name of this call or add validation to make sure this is a Paper enrolment.

Reply:
Renamed, just as to why I did that: since it was only called from the `AutomaticAdjudicationRule` for paper enrolment after verifying that the `paperEnrolmentId` is indeed a paper enrolment id, I thought checking for "NOBCSC" would be redundant.

Reply:
I guess it could also be moved to the EnrolleesService, but not really too critical. The thought was so that if down the road someone needs to check this that they don't overlook this method due to it's name. As well as names should do the best to reflect the functionality.

Reply:
But it sounds like this method isn't needed since you have the enrollee object so this will all turn to dust